docs/docs/resources/architecture/components-hierarchy.md

```diff
@@ -0,0 +1,18 @@
+<!-- I dislike how the mermaid diagram looks, hence I drew it in draw.io. ~Ivan -->
+
```

Comment:
Needed? 😄

Reply:
haha not anymore, I'll keep the mermaid one, would ease things in the future